inbound-whop-app/app/components/email-detail.tsx

```
@@ -0,0 +1,179 @@
+                <span className="font-medium">Attachments:</span> {email.attachments.length} file(s)
+              </div>
+            )}
+          </div>
+        </div>
+
+        {/* Email Content */}
+        <div className="p-6">
+          {email.htmlBody ? (
+            <div 
+              className="prose max-w-none"
+              dangerouslySetInnerHTML={{ __html: email.htmlBody }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)